File: Listas de Exercícios (Python)/Python/Lista 1 - Sala/6.py
print("Prestação atrasada\n")

print("--Valores decimais devem ser digitados com (.)!--")

V=float(input("Valor: "))
T=float(input("Tempo de atraso (unidade de tempo em que a Taxa é cobrada): "))
Ta=float(input("Taxa (em forma de número, exemplo: 90% --> 1.9): "))

P=V+(V*(Ta/100)*T); P="{:.2f}".format(P)

print(f"\nPrestação: R${P}\n")

File: Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 1 - Sala/6.c
#include <stdio.h>

int main()
{
    float T, V, P, TA;
    
    printf("Prestação atrasada\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Valor: "); scanf("%f", &V);
    printf("Tempo de atraso (unidade de tempo em que a Taxa é cobrada): "); scanf("%f", &T);
    printf("Taxa (em forma de número, exemplo: 90% --> 1.9): "); scanf("%f", &TA);
    
    P=V+(V*(TA/100)*T);
    
    printf("\nPrestação: R$%.2f\n", P);
}